src/web/aws-cloudwatch/util.js

```diff
@@ -0,0 +1,11 @@
+const DEFAULT_SETTINGS = { /* Default Advanced Values */
+  awsCloudWatchGlobalInput: '',
+  awsCloudWatchAssumeARN: '',
+  awsCloudWatchBatchSize: '10000',
+  awsCloudWatchThrottleEnabled: '',
+  awsCloudWatchThrottleWait: '1000',
+};
+
+export { DEFAULT_SETTINGS };
+
+export default {};
```

Comment:
I think we can safely make `DEFAULT_SETTINGS` the default export and use the file solely for it.